terraform/spec/dependabot/terraform/file_parser_spec.rb

```diff
@@ -1064,7 +1064,7 @@
       it "returns the correct package manager" do
         expect(package_manager.name).to eq "terraform"
         expect(package_manager.requirement).to be_nil
-        expect(package_manager.version.to_s).to eq "1.10.5"
+        expect(package_manager.version.to_s).to eq "1.11.0"
```

Comment:
Please can we remove this line?
```suggestion
```

Reply:
I would be fine with it, but the change was introduced with the update to Terraform 1.10.5 and I don't know if there was a specific reason to add it.

Reply:
We're moving to this mode going forward to reduce maintainability costs